### PR TITLE
Add sigma to OrderedProbit fixes #5103

### LINKED ISSUE #5103: pm.OrderedProbit bug
So the new `pm.OrderedProbit` looks great. I've been trying to work through some examples and I've noticed a mismatch between the docstring and the working of the function. In the docstring it states that you can provide a `sigma` kwarg (i.e. "The standard deviation of probit function"). However when trying to do a simple example, this errors out. Example...

```python
import pymc3 as pm
import seaborn as sns

samples = pm.OrderedProbit.dist(cutpoints=[-2, 0, 1, 2], eta=0, sigma=1).random(size=500)
sns.countplot(x=samples, color='C0');
```

but it errors with
```
TypeError: __init__() got an unexpected keyword argument 'sigma'
```

So basically the `pm.OrderedProbit` doesn't yet do what it states in the docstring. Taking a look at the code, it looks like the function does not have `sigma` as a kwarg and doesn't do anything with `sigma`.

Version details...
```
Python implementation: CPython
Python version       : 3.9.7
IPython version      : 7.28.0

theano: 1.1.2
xarray: 0.19.0

pandas    : 1.3.4
theano    : 1.1.2
arviz     : 0.11.4
pymc3     : 3.11.2
seaborn   : 0.11.2
matplotlib: 3.4.3
numpy     : 1.21.3
```

### PR DESCRIPTION
**Pull request by zaxtax**
This is to fix OrderedProbit to actually use the sigma argument.

### CODE REVIEW

File: pymc/distributions/discrete.py [hunk 1]
```diff
@@ -1761,16 +1761,16 @@ class _OrderedProbit(Categorical):
     rv_op = categorical
 
     @classmethod
-    def dist(cls, eta, cutpoints, *args, **kwargs):
+    def dist(cls, eta, cutpoints, sigma, *args, **kwargs):
```
Comment (ricardoV94) [inline pymc/distributions/discrete.py]:
```suggestion
    def dist(cls, eta, cutpoints, sigma=1, *args, **kwargs):
```
To make it optional

Comment (zaxtax) [reply pymc/distributions/discrete.py]: Yep, also going to update that and the tests.

**Comment by zaxtax** [comment]
Should be good to go.

File: pymc/distributions/discrete.py [hunk 2]
```diff
@@ -1816,12 +1816,12 @@ class OrderedProbit:
         The length K - 1 array of cutpoints which break :math:`\eta` into
         ranges. Do not explicitly set the first and last elements of
         :math:`c` to negative and positive infinity.
+    sigma: float, default 1.0
+         The standard deviation of probit function.
```
Comment (twiecki) [inline pymc/distributions/discrete.py]:
```suggestion
         Standard deviation of the probit function.
```

**Review by twiecki — APPROVED** [review]

**Comment by twiecki** [comment]
Thanks @zaxtax, good to see you here!

**Pull request merged**